components/calendar/src/cal/chinese.rs

```diff
@@ -2398,6 +2399,58 @@ mod test {
         }
     }
 
+    #[test]
+    #[ignore] // network
+    fn test_against_kasi_data() {
+        use crate::{
+            cal::{Gregorian, LunarChinese},
+            types::MonthCode,
+            Date,
+        };
+
+        // TODO: query KASI directly
```

Comment:
question: is this an actual goal, given that KASI doesn't have a real API for this?

Reply:
- KASI does have an API for this
- this code could scrape the website